src/schema/__tests__/causality_jwt.test.js

```diff
@@ -127,13 +127,21 @@ describe("CausalityJWT", () => {
     })
   })
 
-  it("does not allow a non-admin user or user not associated with sale partner to be operator", () => {
+  it("does not allow a non-admin user or user not associated with sale partner to be operator", done => {
```

Comment:
@yuki24 There are a fair number of tests here that depend upon `runAuthenticatedQuery`? Should all of there `it` expressions be updated to the `it("...", done) => {})` form? If so, while we're here, I'd recommend updating those tests to improve the robustness of MP.

Reply:
Jest resolves the promise if the test returns a Promise object (so `return` is required) so we don't have to add `done()` to every spec. In this case, the catch block doesn't get invoked entirely unless an error is thrown. The extra `done` was added to prevent the test from passing with no expectations. I've replaced the `done` with `expect.assertions(1)` so the test will fail when the operation being tested does not throw an error and I think this is more explicit than `done`.